src/Elementor/Widgets/CommentList/WidgetCommentList.php

```diff
@@ -2124,25 +2143,25 @@ public function renderRatings () {
      *
      * Written in PHP and used to generate the final HTML.
      *
+     * @return void
      * @since 0.1.0
      *
-     * @return void
      */
     protected function render() {
         $settings = $this->get_settings_for_display();
 
-        if ( $settings[ 'comments_ratings_toggle' ] == '' ) {
+        if ( $settings['comments_ratings_toggle'] == '' ) {
             $this->renderComments();
         } else {
             $this->renderRatings();
         }
 
     }
 
-    public static function getCommentContent () {
-        $commentID = isset( $_POST[ 'commentID' ] ) ? $_POST[ 'commentID' ] : false;
+    public static function getCommentContent() {
+        $commentID = filter_has_var( INPUT_POST, 'commentID' ) ? filter_var( INPUT_POST, 'commentID' ) : false;
```

Comment:
Does this even work? `filter_var()` expects parameter 2 to be integer.
I suggest using `filter_input`.

`$commentID = filter_input( INPUT_POST, 'commentID' );`

Reply:
Yes this hasn´t worked. Is fixed now